docs/source/api/apollo-gateway.mdx

```diff
@@ -135,6 +135,41 @@ You can specify any string value for the `name` field, which is used for identif
 **If you _aren't_ using managed federation,** either this field or `supergraphSdl` is required. Do not provide both.
 
 
+</td>
+</tr>
+
+<tr>
+<td>
+
+###### `uplinkEndpoints`
+
+`Array<string>`
+</td>
+<td>
+
+A list of Apollo Uplink URLs the gateway uses to poll for its managed configuration. For details and defaults, see [Apollo Uplink](/managed-federation/uplink/).
+
+Provide this field **only** if you are using managed federation.
+
+</td>
+</tr>
+
+
+<tr>
+<td>
+
+###### `uplinkMaxRetries`
+
+`number`
+</td>
+<td>
+
+The maximum number of times the gateway retries a failed poll request to Apollo Uplink, cycling through its list of [`uplinkEndpoints`](#uplinkendpoints).
+
+The default value is three times your number of `uplinkEndpoints`.
```

Comment:
Might add that there are 2 endpoints by default, which should be the overwhelming majority of cases.